mailpoet/assets/js/src/automation/testing.tsx

```diff
@@ -17,7 +17,8 @@ const createDelayStep = (nextStepId: string) => ({
   key: 'core:delay',
   next_step_id: nextStepId,
   args: {
-    seconds: 60,
+    delay: 1,
+    delay_type: 'HOURS',
```

Comment:
So, when I would have 3600 seconds it would display 1 hour?

My problem would be: And if someone types in 7 days it would switch to 1 week? That likely feels strange and could lead to wrong inputs, when someone who typed in 7 days sees suddenly a 1 in the input field (without seeing the day switched to week) and changes it back to 7.

Reply:
Ah, right. We need to store it to display the UI correctly 👍